package.json

```diff
@@ -42,10 +42,25 @@
     "format-php": "composer format",
     "phpstan": "composer phpstan",
     "lint-php": "composer lint:all",
-    "test-php": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test",
-    "test-php-multisite": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test-multisite",
-    "test-php-plugins": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test:plugins",
-    "test-php-multisite-plugins": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test-multisite:plugins",
+    "test:php": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance composer phpunit",
+    "test-php-plugins": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance/plugins/$npm_config_plugin env WP_MULTISITE=$npm_config_multisite php ../../vendor/bin/phpunit --verbose",
```

Comment:
Does this cmd work for you?

Reply:
This non-user-facing command is extended by commands like `test:php:auto-sizes` etc. In case someone wants to use it directly, a `--plugin` flag should be passed e.g. `npm run test-php-plugins --plugin=auto-sizes`.

Reply:
My thought process here is to use `-` in non-user-facing commands and use `:` separator in user-facing commands. Please LMK if it makes sense.

Reply:
This set of commands is getting pretty verbose. I think it could be easier easier to have a single bash file or node CLI script to run these tests, which can accept different arguments for a single plugin or configuration (e.g., multisite) rather than needing to maintain separate scripts for each plugin here.